docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -225,6 +228,18 @@ property definition to the ``_meta.presto`` property of the target index mapping
         }
     }'
 
+Date types
+^^^^^^^^^^
+
+Elasticsearch supports a wide array of `date`_ formats including `built-in date formats`_ and also `custom date formats`_.
```

Comment:
Wrap at 80 char width please

Reply:
I have wrapped it here now. But the same convention hasn't been followed throughout the document.

Reply:
I know .. we will have to do a bulk wrap edit at some stage.. for now we have been doing piecemeal improvements so doing it for new content helps. And also works better for reviews..